src/ext/TagExtension/wixext/Xsd/tag.xsd

```diff
@@ -58,30 +78,27 @@
                   It is recommended to <html:strong>not</html:strong> set this attribute.</xs:documentation>
               </xs:annotation>
           </xs:attribute>
-          <xs:attribute name="Licensed" type="YesNoTypeUnion">
+          <xs:attribute name="Licensed" type="xs:string">
             <xs:annotation>
-              <xs:documentation>Indicates whether the software requires a license. The default is
-              "no". </xs:documentation>
+              <xs:appinfo>
+                <xse:deprecated />
+              </xs:appinfo>
+            </xs:annotation>
+          </xs:attribute>
+          <xs:attribute name="Type" type="xs:string">
+            <xs:annotation>
+              <xs:appinfo>
+                <xse:deprecated />
+              </xs:appinfo>
             </xs:annotation>
           </xs:attribute>
-        <xs:attribute name="Type" type="TagTypeUnion">
```

Comment:
`Type` is still present. Its type was changed to string (to be allowed to remove the union) and marked deprecated. Just like `Licensed`.